web/themes/custom/pineapple/sass/components/navigation/_main-menu.scss

```diff
@@ -137,3 +137,81 @@
     }
   }
 }
+
+.footer {
+  display: flex;
+  flex: 0 0 33.33%;
+  background-color: $primary;
+  margin-top: $unit * 2;
```

Comment:
Make this `$unit * 4` and also add the same amount of margin bottom on the hero container

Reply:
done